src/views/catalog/customize/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const VIRTUAL_MACHINE_KIND = 'VirtualMachine';
```

Comment:
this const is redundant, please import `VirtualMachineModel` like this:
`import VirtualMachineModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineModel';`
and use the kind property instead of `VIRTUAL_MACHINE_KIND` const

Reply:
okay thanks